src/app/(home)/aboutUs/page.tsx

```diff
@@ -0,0 +1,20 @@
+import NavBar from "../../../components/userProfileLayout/navBar/navBar";
+import UserProfile from "../../../components/userProfileLayout/userProfile/userProfile";
+import UserProfileHeader from "../../../components/userProfileLayout/userProfileHeader/userProfileHeader";
+
+import styles from "./aboutUs.module.scss";
+function AboutUs() {
+	return (
+		<div className={styles.aboutUsWrap}>
+			<div className={styles.aboutUsWrap__navBar}>
+				<NavBar />
+			</div>
+			<div className={styles.aboutUsProfile}>
+				<UserProfileHeader />
+				<UserProfile />
```

Comment:
<div>
					<div>
						Content
					</div>
					<UserProfile />
				</div>
can you move it from aboutUs page to app/(privateProfile)/page.tsx?